docs/reference/commandline/trust_key_load.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "key-load"
+description: "The key-load command description and usage"
+keywords: "Key, notary, trust"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# trust key-load
+
+```markdown
+Usage:  docker trust key-load KEY [KEY...]
+
+Load a signing key
```

Comment:
Just leaving a note here that we might want to update this once we address the feedback from today